clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/provider/view/OpenstackLoadBalancerProvider.groovy

```diff
@@ -134,9 +136,12 @@ class OpenstackLoadBalancerProvider implements LoadBalancerProvider<OpenstackLoa
   // TODO: OpenstackLoadBalancerSummary is not a LoadBalancerProvider.Item, but still
   // compiles anyway because of groovy magic.
   List<OpenstackLoadBalancerSummary> list() {
-    getLoadBalancers('*', '*', '*').collect { lb ->
-      new OpenstackLoadBalancerSummary(account: lb.account, region: lb.region, id: lb.id, name: lb.name)
-    }.sort { it.name }
+    log.info("LoadBalanceProvider:list() start")
+    def searchKey = Keys.getLoadBalancerKey('*', '*', '*', '*');
+    Collection<String> identifiers = cacheView.filterIdentifiers(LOAD_BALANCERS.ns, searchKey)
+    def result = getSummaryForLoadBalancers(identifiers).values() as List
+    log.info("LoadBalanceProvider:list() end")
```

Comment:
right....those were for debugging....im going to take those out today and fix the 2 tests that are failing which part of is irrelevant